common/src/main/java/com/microsoft/identity/common/internal/util/AndroidKeyStoreUtil.java

```diff
@@ -447,11 +449,27 @@ public static synchronized SecretKey unwrap(@NonNull final byte[] wrappedKeyBlob
                 exception
         );
         if (exception instanceof InvalidKeyException) {
+            final android.security.KeyStoreException keyStoreException = findKeyStoreException(exception);
```

Comment:
>KeyStoreException [](http://example.com/codeflow?start=35&length=17)

Seems this object >= 33. in this line it used without protection and can cause NoClassDefError exception

Refactor such that all references to this android.security.Exception are API safe.